modules/end-user-guide/partials/assembly_authoring-devfiles-version-2.adoc

```diff
@@ -1,21 +1,34 @@
 :parent-context-of-authoring-devfiles-version-2: {context}
 
 [id="authoring-devfiles-version-2_{context}"]
-= Authoring devfiles version 2
+= Authoring a devfile 2.0.0
```

Comment:
I believe Che always follows the latest version of the devfile 2 specifications, so putting the minor and patch version here is not a feature, we should keep reference to "devfile 2". @l0rd can you confirm?